Update Catalogs doc - EA - 4/19 (#3450)
braze-inc/braze-docs: _docs/_hidden/private_betas/catalog.md

Once this information is imported, you can begin accessing it in messages in a similar way to accessing custom attributes or custom event properties through Liquid.

{% alert important %}
Catalogs are currently in early access. Contact your Braze account manager if you are interested in participating in the early access.
{% endalert %}

> If you'd like to share your feedback on this feature or make a request, contact the <a href="mailto:catalogs-product@braze.com">Braze Catalogs team</a>!

Reviewer: You can do this with markdown! Also, I've received feedback on other docs that people tend to prefer having the email spelled out as the link, so that they can copy+paste into their email client if they don't use the OS default.

```suggestion
If you'd like to share your feedback on this feature or make a request, contact the Braze Catalogs team at [catalogs-product@braze.com](mailto:catalogs-product@braze.com).
```

| Characters limit for item column name | Up to 250 characters | Up to 250 characters |
{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}

> Interested in upgrading your experience to Catalog Pro? Contact the <a href="mailto:catalogs-product@braze.com">Braze Catalogs team</a> for more information.

Reviewer: ```suggestion
Interested in upgrading your experience to Catalog Pro? Contact the Braze Catalogs team at [catalogs-product@braze.com](mailto:catalogs-product@braze.com) for more information.
```
